### DISCUSSION TITLE
So I just created a theme for the splash application..

### DISCUSSION THREAD

**Discussion by tom-avilius** [catppuccin/catppuccin#2186] [General]
Is this worth adding to the showcase..?

It is like a desktop theme similar to rainmeter ones but uses html, css and js.

![Demo](https://raw.githubusercontent.com/tom-avilius/hersweetkiss.github.io/master/assets/img/S6.jpg)

![Demo](https://raw.githubusercontent.com/tom-avilius/hersweetkiss.github.io/master/assets/img/S5.jpg)

![Demo](https://raw.githubusercontent.com/tom-avilius/hersweetkiss.github.io/master/assets/img/S4.jpg)

**Comment by isabelroses** [comment]
> P.S. The images are not being displayed in the markdown preview, not sure why. The link is correct.

You should use the raw.githubuser links to get the images to display

```md
![example](https://raw.githubusercontent.com/tom-avilius/hersweetkiss.github.io/master/assets/img/S6.jpg)
```

**Reply by tom-avilius** [reply]
Thankyou, this works.. :)

**Comment by sgoudham** [comment]
Hey :wave:, as stated in the [README Showcase section](https://github.com/catppuccin/catppuccin?tab=readme-ov-file#-showcase), we aren't accepting personal configurations / desktop customisations to the showcase - will be closing now. (nice job on the customisation though!)
